finality-aleph/src/new_network/connection_manager/session.rs

```diff
@@ -317,6 +317,18 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn handler_without_node_index_nor_authority_pen_returns_none_for_authentication() {
```

Comment:
I know I started with the horribly long test names, but maybe here it could just be `non_validator_handler_returns...`? :P